.github/workflows/stage-dataset.yml

```diff
@@ -10,23 +10,54 @@ on:
         type: boolean
         description: "Set to true to skip the file-info upload (save some processing time)"
         default: false
+      database:
+        description: "Please choose a database to release dataset"
```

Comment:
I think we should change the word `release` here, as we are not releasing anything.  Maybe `upload` is a better word choice?